android-agent/src/main/java/io/opentelemetry/android/OpenTelemetryRumImpl.java

```diff
@@ -19,7 +18,7 @@ final class OpenTelemetryRumImpl implements OpenTelemetryRum {
     }
 
     @Override
-    public OpenTelemetry getOpenTelemetry() {
+    public OpenTelemetrySdk getOpenTelemetry() {
```

Comment:
I don't like having to use `OpenTelemetrySdk` instead of `OpenTelemetry`, though it was needed because of the way that the existing crash instrumentation works, as it needs to call `SdkLoggerProvider.forceFlush()` inside `CrashReportingExceptionHandler.uncaughtException` to try and get the crash data exported before the app is closed. I hope it's something we can improve later.

Reply:
Never mind, it seems like just casting it to the SDK impl for the crash instrumentation does the trick without any side effects I could spot.